internal/accounts/create.go

```diff
@@ -134,3 +148,258 @@ func create(
 		include: createFlags.Include,
 	}, nil
 }
+
+func createInteractive(state *flowkit.State, loader flowkit.ReaderWriter) (*flow.Account, error) {
+	network := output.CreateAccountNetworkPrompt()
+	log := output.NewStdoutLogger(output.InfoLog)
+
+	// create new gateway based on chosen network
+	gw, err := gateway.NewGrpcGateway(network.Host)
+	if err != nil {
+		return nil, err
+	}
+	service := services.NewServices(gw, state, output.NewStdoutLogger(output.NoneLog))
+	log.Info("\nSTARTING ACCOUNT CREATION PROCESS: ")
+	log.Info("\n------------------------")
+	log.Info(fmt.Sprintf("%s (1/3) Generate public and private keys", output.GoEmoji()))
+	log.Info(fmt.Sprintf("%s (2/3) Create account on %s with generated keys", output.GoEmoji(), network.Name))
+	log.Info(fmt.Sprintf("%s (3/3) Save newly created account on flow.json", output.GoEmoji()))
+	log.Info("\n------------------------")
+	time.Sleep(time.Second * 2)
```

Comment:
Improved interactivity by detailing each step that is happening and adding time lag in between steps to allow users to have "the feel of interactivity"